paws.common/R/config.R

```diff
@@ -95,6 +98,41 @@ check_config_file_region <- function() {
   return(region)
 }
 
+# Tries to get the region from the config file
+check_config_file_assume_role <- function() {
+
+  config_path <- file.path(get_aws_path(), "config")
+
+  if (!file.exists(config_path)) return(NULL)
+  
+  aws_profile <- get_profile_name()
+
+  config_values <- ini::read.ini(config_path)
+
+  if (is.null(config_values[[aws_profile]])) {
+    aws_profile <- paste("profile", aws_profile)
+    config_values <- ini::read.ini(config_path)
+    if (is.null(config_values[[aws_profile]])) return(NULL)
+  }
+
+  role_arn <- config_values[[aws_profile]]$role_arn
+  source_profile <- config_values[[aws_profile]]$source_profile
+  role_session_name <- config_values[[aws_profile]]$role_session_name
+  
+  if (is.null(role_session_name)) {
+    role_session_name <- paste0(sample(LETTERS, 15, replace = TRUE),
```

Comment:
This needs a comment -- not sure why this is random like this.

Reply:
I added a comment. This is because a role session name is required but may not always be specified beforehand. Therefore this creates one for you